__tests__/app.test.js

```diff
@@ -43,9 +43,28 @@ describe('03_separation-of-concerns-demo routes', () => {
     const order = await Order.insert({ quantity: 10 });
     
     return request(app)
-    .get('/api/v1/orders')
-    .then((res) => {
-      expect(res.body).toEqual([order]);
+      .get('/api/v1/orders')
+      .then((res) => {
+        expect(res.body).toEqual([order]);
       });
   });
+
+  it('update a single order', async () => {
+    const order = await Order.insert({
+      quantity: 10 });
+    
+    const res = await request(app)
+      .put(`/api/v1/orders/${order.id}`)
+      .send({ quantity: 2 });
+    expect(res.body).toEqual({ ...order });
```

Comment:
There's some weirdness going on with your put route/update methods that are allowing this test to pass; logically, this test should fail. You've got `order`, which starts with a quantity of 10, and then you make a put request where you attempt to update the quantity to 2. If it were successful, your last line would fail because you're expecting it to equal the original order instead of `{ ...order, quantity: 2 }`.

I'll add comments to the relevant parts of the route to show what's going on.